typescript/api/controllers/webservice/RecordController.ts

```diff
@@ -256,6 +257,22 @@ export module Controllers {
         });
     }
 
+    public getRecordAudit(req, res) {
+      const brand = BrandingService.getBrand(req.session.branding);
+      var oid = req.param('oid');
+
+      Observable.fromPromise(this.RecordsService.getRecordAudit(oid)).subscribe(audit => {
```

Comment:
Once you've done the change in https://github.com/redbox-mint/sails-hook-redbox-storage-mongo/pull/2
Adding support for the date range filtering would be useful in the api